### PR TITLE
more clarity on the complexity of the check in TS.ADD

### PR DESCRIPTION
**Pull request by K-Jo**

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ Optional args:
    * labels - set of key-value pairs that represent metadata labels of the key
 
 #### Notes
-You can use this command to add data to an non existing timeseries in a single command.  This is the reason why the labels and retentionsecs are optional arguments.  When specified, RedisTimeSeries will check if it needs to update the labels and or retentionSecs which introduces additional complexity.
+You can use this command to add data to an non existing timeseries in a single command.  This is the reason why the labels and retentionsecs are optional arguments.  When specified, RedisTimeSeries will check if it needs to update the labels and or retentionSecs.  This check, next to the update itself, introduces additional time complexity.
```
Comment (markusr) [inline docs/commands.md]: You should add:
```
If this command is used to add data to an existing timeseries the optional parameters are ignored. 
```

**Comment by danni-m** [comment]
@K-Jo the docs says that it will update the key labels, which is incorrect. can we update to reflect that this is only for non existing keys?

**Comment by K-Jo** [comment]
@markusr is this ok?  Happy to merge it in

**Comment by markusr** [comment]
@K-Jo Yes. LGTM.

**Pull request merged**